src/storage/v2/edge_accessor.cpp

```diff
@@ -207,19 +207,20 @@ Result<PropertyValue> EdgeAccessor::GetProperty(PropertyId property, View view)
   if (!storage_->config_.items.properties_on_edges) return PropertyValue();
   bool exists = true;
   bool deleted = false;
-  PropertyValue value;
```

Comment:
RVO doesn't do anything here?

Reply:
Maybe. But regardless of that without LTO, the cost of `PropertyValue::operator=(PropertyValue &&)` is noticeable and hence we want to avoid it. Hence the `emplace` on line 216 to do move construction inside an optional.

Maybe PropertyValue should be header only? But that can be done in another PR if needed.